library/private/options_tools.h.in

```diff
@@ -78,7 +110,7 @@ int parse(const std::string& str)
   // TODO implement more parsing possibilities ?
   try
   {
-    return std::stoi(str);
+    return parse_int(str);
```

Comment:
supports NaN and Inf ?

Reply:
there's no way to properly encode that as `int` tho, is there? I guess would could fake `±inf` with some kind of `MIN_INT`/`MAX_INT` but is it worth it?

Reply:
let's wait until we have a use-case